common/src/main/java/com/wynntils/services/map/Label.java

```diff
@@ -6,19 +6,33 @@
 
 import com.wynntils.utils.StringUtils;
 import com.wynntils.utils.mc.type.Location;
+import com.wynntils.utils.type.RangedValue;
 
 public class Label {
     private final String name;
     private final int x;
     private final int z;
     private final int layer;
-    private final String level;
 
-    public Label(String name, int x, int z, int layer, String level) {
+    // level is provided as a string for some data sources
```

Comment:
Can we get around this and make sure we only use integer values? I think the source of this was the Wynncraft map label thing, but that is apparently very hand-crafted, and if they write bad data we can adjust it, I think. I know I was very keen in old times to make sure we match the official data exactly, but my opinion has changed in that case, and I think now it is better if we provide a curated solution.